modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4249,7 +4249,13 @@ else if (!(number instanceof Long)) {
 			values.put(name, number);
 		}
 		else if (javaTypeClass == String.class) {
-			values.put(name, (String)object);
+			String string = (String)object;
+
+			if (string == null) {
+				string = StringPool.BLANK;
+			}
+
+			values.put(name, string);
```

Comment:
@vngoliveira it feels like a breaking change. Please open a thread to discuss this solution with the team

Reply:
[slack thread](https://liferay.slack.com/archives/C03S91MJG6L/p1736777923039709)